exasol_transformers_extension/resources/templates/question_answering_udf.jinja.sql

```diff
@@ -1,11 +1,13 @@
 CREATE OR REPLACE {{ language_alias }} SET SCRIPT "TE_QUESTION_ANSWERING_UDF"(
+    device_name VARCHAR(2000000),
```

Comment:
I suggest using an integer. Null means CPU, positive int means the particular device